components/ui/AppFiltersResultsLayout.vue

```diff
@@ -29,21 +29,38 @@ export default class AppFiltersResultsLayouts extends Vue {
 }
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
 .app-filters-results-layout {
-  &__main-section {
+  ::v-deep &__main-section {
     @include mq($until: medium) {
       margin-top: $layout-04;
     }
   }
 
-  &__results-item {
+  ::v-deep &__results-item {
     margin-bottom: $layout-02;
 
     @include mq($until: large) {
       margin-bottom: $layout-01;
     }
+
+    @include mq($from: medium) {
```

Comment:
At least at the moment, the card's height should be adapted to the content, so please, delete these `min-height`